src/routes/forgot-password/forgot-password-container.js

```diff
@@ -1,19 +1,19 @@
 import { connect } from 'react-redux';
 import { ForgotPassword } from './forgot-password';
 import { push } from 'react-router-redux';
-import { resolveUser } from 'redux/actions';
+import { resetPassword } from 'redux/actions';
 
-const mapStateToProps = (state) => ({
+/* const mapStateToProps = (state) => ({
   user: state.user.user
-});
+}); */
 
 const mapDispatchToProps = dispatch => ({
   push: (path) => dispatch(push(path)),
-  resolveUser: () => dispatch(resolveUser())
+  resetPassword: () => dispatch(resetPassword())
 });
 
 export const ForgotPasswordContainer =
   connect(
-    mapStateToProps,
+    undefined, // mapStateToProps,
```

Comment:
why undefined?

Reply:
It does not need any state.
I *think* that one then sets it to undefined.
But I should perhaps look that up to be sure.

Reply:
https://github.com/reactjs/react-redux/blob/master/docs/api.md#arguments

"[mapStateToProps(state, [ownProps]): stateProps] (Function): *If* this argument is specified, the new component will [...] If you don't want to subscribe to store updates, pass null or undefined in place of mapStateToProps."